Action Buttons Bar implemented

sunny9393 opened the pull request.

sunny9393 commented:
Hello, 

I have committed my code changes. Please have a look at the pull request.

Branch Name: sunny9393:feat/action-buttons-bar

igodorogea commented:
Hi @sunny9393,
thank you for quick reply.
However this implementation is not what I'm expecting.
Please keep in mind that this challenge is to prove your advanced skills, like:
 - complex content projection with custom directives
 - application structure decisions (reusable component / SCAM pattern)
 - at least an attempt to solve requirement 2.i.
 - a usable implementation requirement 2.ii.

Please give it one more try and let me know if you have any questions.

sunny9393 commented:
Hi Ivan,

Can we schedule a call, I wanted to understand what am I missing.

I have already added these two points into code which you have mentioned

   - at least an attempt to solve requirement 2.i. (you can drag and drop
   this bar anywhere in the browser as shown in video)
   - a usable implementation requirement 2.ii.

   [image: image.png]

   Thanks
   Sunny


On Wed, Apr 6, 2022 at 4:26 AM Ivan ***@***.***> wrote:

> Hi @sunny9393 <https://github.com/sunny9393>,
> thank you for quick reply.
> However this implementation is not what I'm expecting.
> Please keep in mind that this challenge is to prove your advanced skills,
> like:
>
>    - complex content projection with custom directives
>    - application structure decisions (reusable component / SCAM pattern)
>    - at least an attempt to solve requirement 2.i.
>    - a usable implementation requirement 2.ii.
>
> Please give it one more try and let me know if you have any questions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/igodorogea/angular-interview-challenge/pull/1#issuecomment-1089986908>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHIJIZYM62OGXET2JCQ5MNLVDVDBZANCNFSM5SUILWFA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sunny9393 commented:
Hi Ivan,

I have added content projection with custom directives to the code and
created new Action Bar

It also has "drag and Drop" & "Expand/Collapse" feature as shown in
sample video. Please merge the code to main branch or access the feature
branch("feat/action-buttons-bar") and try it in localhost.

I have attached the screenshot as well.

Thanks & Regards
Sunny Parikh
[image: image.png]

On Wed, Apr 6, 2022 at 9:27 AM sunny parikh ***@***.***>
wrote:

> Hi Ivan,
>
> Can we schedule a call, I wanted to understand what am I missing.
>
> I have already added these two points into code which you have mentioned
>
>    - at least an attempt to solve requirement 2.i. (you can drag and drop
>    this bar anywhere in the browser as shown in video)
>    - a usable implementation requirement 2.ii.
>
>    [image: image.png]
>
>    Thanks
>    Sunny
>
>
> On Wed, Apr 6, 2022 at 4:26 AM Ivan ***@***.***> wrote:
>
>> Hi @sunny9393 <https://github.com/sunny9393>,
>> thank you for quick reply.
>> However this implementation is not what I'm expecting.
>> Please keep in mind that this challenge is to prove your advanced skills,
>> like:
>>
>>    - complex content projection with custom directives
>>    - application structure decisions (reusable component / SCAM pattern)
>>    - at least an attempt to solve requirement 2.i.
>>    - a usable implementation requirement 2.ii.
>>
>> Please give it one more try and let me know if you have any questions.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/igodorogea/angular-interview-challenge/pull/1#issuecomment-1089986908>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AHIJIZYM62OGXET2JCQ5MNLVDVDBZANCNFSM5SUILWFA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
